tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -0,0 +1,202 @@
+apiVersion: cinder.openstack.org/v1beta1
+kind: Cinder
+metadata:
+  name: cinder
+  namespace: openstack
+spec:
+  serviceUser: cinder
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: cinder
+  rabbitMqClusterName: rabbitmq
+  cinderAPI:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-api:current-tripleo
+  cinderScheduler:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-scheduler:current-tripleo
+  cinderBackup:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-backup:current-tripleo
+  secret: cinder-secret
+  cinderVolumes:
+    volume1:
+      containerImage: quay.io/tripleozedcentos9/openstack-cinder-volume:current-tripleo
+      replicas: 1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cinder
+  namespace: openstack
+  ownerReferences:
+  - apiVersion: cinder.openstack.org/v1beta1
+    blockOwnerDeletion: true
+    controller: true
+    kind: CinderAPI
+    name: cinder-api
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      service: cinder
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        service: cinder
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: service
+                  operator: In
+                  values:
+                  - cinder
+              topologyKey: kubernetes.io/hostname
+            weight: 1
+      containers:
+      - args:
+        - -c
+        - /usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start
+        command:
+        - /bin/bash
+        env:
+        - name: CONFIG_HASH
```

Comment:
I'm not sure how this hash is generated but it probably changes between runs, will make the tests fail

Reply:
ack, removed